checkov/sast/checks_infra/base_registry.py

```diff
@@ -24,7 +25,10 @@ def __init__(self, checks_dir: str, temp_semgrep_rules_path: str | None = None)
         self.rules: List[Dict[str, Any]] = []
         self.checks_dir = checks_dir
         self.logger = logging.getLogger(__name__)
-        self.parser = SastCheckParser()
+        self.parsers = {
+            BqlVersion.V0_1.value: SastCheckParserV01(),
```

Comment:
do we actually need BQL1?
as Rotem once said - git history remembers

Reply:
Good q, We deleted the policies themselves as its easy to restore from history, I think the parser would be more complicated to restore if we want as it would have conflicts with time, but can also remove it if we want, really maybe not needed. Will ask rotem and we can delete in a following PR